SERIOUS ERROR WITH MYSQL

After you updated the jar my server no longer connects to mysql, I tried another fork and it works. Please settle down as soon as possible!

gives me these errors in console:
```
java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql://127.0.0.1:3306/database1
	at com.andrei1058.bedwars.libs.hikari.util.DriverDataSource.<init>(DriverDataSource.java:114) ~[?:?]
	at com.andrei1058.bedwars.libs.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:325) ~[?:?]
	at com.andrei1058.bedwars.libs.hikari.pool.PoolBase.<init>(PoolBase.java:114) ~[?:?]
	at com.andrei1058.bedwars.libs.hikari.pool.HikariPool.<init>(HikariPool.java:108) ~[?:?]
	at com.andrei1058.bedwars.libs.hikari.HikariDataSource.<init>(HikariDataSource.java:81) ~[?:?]
	at com.andrei1058.bedwars.database.MySQL.connect(MySQL.java:87) ~[?:?]
	at com.andrei1058.bedwars.BedWars.onEnable(BedWars.java:353) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321) ~[server.jar:git-NachoSpigot-"4839ac6"]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:332) [server.jar:git-NachoSpigot-"4839ac6"]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:393) [server.jar:git-NachoSpigot-"4839ac6"]
	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.java:374) [server.jar:git-NachoSpigot-"4839ac6"]
	at org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.java:333) [server.jar:git-NachoSpigot-"4839ac6"]
	at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:205) [server.jar:git-NachoSpigot-"4839ac6"]
	at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:559) [server.jar:git-NachoSpigot-"4839ac6"]
	at java.base/java.lang.Thread.run(Thread.java:829) [?:?]
Caused by: java.sql.SQLException: No suitable driver
	at java.sql/java.sql.DriverManager.getDriver(DriverManager.java:298) ~[java.sql:?]
	at com.andrei1058.bedwars.libs.hikari.util.DriverDataSource.<init>(DriverDataSource.java:106) ~[?:?]
	... 14 more
```

```
[14:03:18 INFO]: Patching Citizens, please wait.
>WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.bukkit.craftbukkit.libs.javassist.util.proxy.SecurityActions (file:/my/directory/server.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of org.bukkit.craftbukkit.libs.javassist.util.proxy.SecurityActions
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[14:03:18 INFO]: Successfully patched Citizens!
```

--- a/NachoSpigot-Server/pom.xml
+++ b/NachoSpigot-Server/pom.xml
@@ -84,7 +84,7 @@
         <dependency>
             <groupId>org.xerial</groupId>
             <artifactId>sqlite-jdbc</artifactId>
-            <version>3.36.0.3</version>
+            <version>3.7.2</version>
             <type>jar</type>
             <scope>runtime</scope>
         </dependency>

